packages/dds/sequence/src/test/attribution/documents/basic/attributor-no-compression-snap.json

```diff
@@ -0,0 +1,289 @@
+{
+    "type": 1,
+    "tree": {
+        "content": {
+            "type": 1,
+            "tree": {
+                "header": {
+                    "type": 2,
+                    "content": "{\"chunkStartSegmentIndex\":0,\"chunkSegmentCount\":23,\"chunkLengthChars\":114,\"totalLengthChars\":114,\"totalSegmentCount\":23,\"chunkSequenceNumber\":76,\"segmentTexts\":[{\"text\":\"Q\",\"props\":{\"prop3\":\"ZCYqA\",\"prop1\":\"8Ykh5\",\"prop2\":\"W44Ny\"}},{\"text\":\"cW\",\"props\":{\"prop3\":\"CErX3\",\"prop1\":\"T17Ub\",\"prop2\":\"W44Ny\"}},\"r\",{\"text\":\"4\",\"props\":{\"prop3\":\"CErX3\",\"prop1\":\"T17Ub\",\"prop2\":\"W44Ny\"}},\"FPJ\",{\"text\":\"J7ao\",\"props\":{\"prop1\":\"SS7fD\",\"prop3\":\"JTwq4\",\"prop2\":\"nM1AA\"}},\"4BEjKWq5P\",{\"text\":\"o52ikqzdD\",\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"}},{\"text\":\"5sQdsKCFyt\",\"props\":{\"prop2\":\"dHWn1\",\"prop1\":\"m4xRB\",\"prop3\":\"5KoWp\"}},{\"text\":\"qT9t\",\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"}},{\"text\":\"uk\",\"props\":{\"prop1\":\"7LqTM\",\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"}},{\"text\":\"Mj6\",\"props\":{\"prop1\":\"T17Ub\",\"prop3\":\"5KoWp\",\"prop2\":\"dHWn1\"}},{\"text\":\"FwbBGFPu2weMKNqi1Z7dFxcuz\",\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"}},\"G\",{\"text\":\"ujj\",\"props\":{\"prop1\":\"4QKwr\",\"prop3\":\"wrDoT\",\"prop2\":\"NhFnu\"}},\"Nqwgn\",{\"text\":\"o\",\"props\":{\"prop1\":\"4QKwr\",\"prop3\":\"wrDoT\",\"prop2\":\"NhFnu\"}},{\"text\":\"T\",\"props\":{\"prop1\":\"4QKwr\",\"prop3\":\"wrDoT\",\"prop2\":\"whoZs\"}},{\"text\":\"LWf3\",\"props\":{\"prop2\":\"whoZs\",\"prop1\":\"28bxC\"}},\"u\",{\"text\":\"QDc7i7Em\",\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"}},{\"text\":\"u\",\"props\":{\"prop2\":\"whoZs\"}},\"2uGpRExau9X3HeT\"],\"attribution\":{\"keys\":[5,14,59,5,43,50,73,68,50,43,61,33,51,52,71,52,51,33,47,66,70,66,56,60,56,64,36,62,36,23,48,69,48,11,75,11,1,40,72,57,34,44],\"posBreakpoints\":[0,1,3,4,5,6,8,12,16,21,22,27,28,29,31,40,44,46,49,53,56,65,66,67,71,73,75,78,83,85,86,89,91,94,95,96,97,98,99,106,109,111],\"length\":114},\"headerMetadata\":{\"orderedChunkMetadata\":[{\"id\":\"header\"}],\"sequenceNumber\":76,\"totalLength\":114,\"totalSegmentCount\":23}}"
```

Comment:
@anthony-murphy FYI I think format changes in the most recent update are going to be what we want, at least for the "merge-tree content -> sequence number" lookup. Here I have them aggregated on a per-chunk basis (so this doesn't break partial load functionality) but still separate from the per-segment list. If we wanted to put them into segments we'd need to introduce some back-compat logic with legacy snapshots because that uses `.toJSONObject` directly in the final format, so there isn't a place to put the attribution information in all cases like there is in V1.